src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -3728,6 +3730,132 @@ public sealed class SocketsHttpHandler_RequestContentLengthMismatchTest_Http3 :
         protected override Version UseVersion => HttpVersion.Version30;
     }
 
+    public abstract class SocketsHttpHandler_SslTest : HttpClientHandlerTestBase
```

Comment:
Nit: SslTest => SslOptionsTest? (If you change it, also change the derived type names)

Reply:
I was not sure. The intention is to test security related functions e.g. SslStream, Quic & certificate handling...